packages/svelte/svelte-html.d.ts

```diff
@@ -0,0 +1,250 @@
+			ellipse: HTMLProps<'ellipse', SVGAttributes>;
+			feBlend: HTMLProps<'feBlend', SVGAttributes>;
+			feColorMatrix: HTMLProps<'feColorMatrix', SVGAttributes>;
+			feComponentTransfer: HTMLProps<'feComponentTransfer', SVGAttributes>;
+			feComposite: HTMLProps<'feComposite', SVGAttributes>;
+			feConvolveMatrix: HTMLProps<'feConvolveMatrix', SVGAttributes>;
+			feDiffuseLighting: HTMLProps<'feDiffuseLighting', SVGAttributes>;
+			feDisplacementMap: HTMLProps<'feDisplacementMap', SVGAttributes>;
+			feDistantLight: HTMLProps<'feDistantLight', SVGAttributes>;
+			feDropShadow: HTMLProps<'feDropShadow', SVGAttributes>;
+			feFlood: HTMLProps<'feFlood', SVGAttributes>;
+			feFuncA: HTMLProps<'feFuncA', SVGAttributes>;
+			feFuncB: HTMLProps<'feFuncB', SVGAttributes>;
+			feFuncG: HTMLProps<'feFuncG', SVGAttributes>;
+			feFuncR: HTMLProps<'feFuncR', SVGAttributes>;
+			feGaussianBlur: HTMLProps<'feGaussianBlur', SVGAttributes>;
+			feImage: HTMLProps<'feImage', SVGAttributes>;
+			feMerge: HTMLProps<'feMerge', SVGAttributes>;
+			feMergeNode: HTMLProps<'feMergeNode', SVGAttributes>;
+			feMorphology: HTMLProps<'feMorphology', SVGAttributes>;
+			feOffset: HTMLProps<'feOffset', SVGAttributes>;
+			fePointLight: HTMLProps<'fePointLight', SVGAttributes>;
+			feSpecularLighting: HTMLProps<'feSpecularLighting', SVGAttributes>;
+			feSpotLight: HTMLProps<'feSpotLight', SVGAttributes>;
+			feTile: HTMLProps<'feTile', SVGAttributes>;
+			feTurbulence: HTMLProps<'feTurbulence', SVGAttributes>;
+			filter: HTMLProps<'filter', SVGAttributes>;
+			foreignObject: HTMLProps<'foreignObject', SVGAttributes>;
+			g: HTMLProps<'g', SVGAttributes>;
+			image: HTMLProps<'image', SVGAttributes>;
+			line: HTMLProps<'line', SVGAttributes>;
+			linearGradient: HTMLProps<'linearGradient', SVGAttributes>;
+			marker: HTMLProps<'marker', SVGAttributes>;
+			mask: HTMLProps<'mask', SVGAttributes>;
+			metadata: HTMLProps<'metadata', SVGAttributes>;
+			mpath: HTMLProps<'mpath', SVGAttributes>;
+			path: HTMLProps<'path', SVGAttributes>;
+			pattern: HTMLProps<'pattern', SVGAttributes>;
+			polygon: HTMLProps<'polygon', SVGAttributes>;
+			polyline: HTMLProps<'polyline', SVGAttributes>;
+			radialGradient: HTMLProps<'radialGradient', SVGAttributes>;
+			rect: HTMLProps<'rect', SVGAttributes>;
+			stop: HTMLProps<'stop', SVGAttributes>;
+			switch: HTMLProps<'switch', SVGAttributes>;
+			symbol: HTMLProps<'symbol', SVGAttributes>;
+			text: HTMLProps<'text', SVGAttributes>;
+			textPath: HTMLProps<'textPath', SVGAttributes>;
+			tspan: HTMLProps<'tspan', SVGAttributes>;
+			use: HTMLProps<'use', SVGAttributes>;
+			view: HTMLProps<'view', SVGAttributes>;
+
+			// Svelte specific
+			'svelte:window': HTMLProps<'svelte:window', HTMLAttributes>;
+			'svelte:body': HTMLProps<'svelte:body', HTMLAttributes>;
+			'svelte:document': HTMLProps<'svelte:document', HTMLAttributes>;
+			'svelte:fragment': { slot?: string };
+			'svelte:options': { [name: string]: any };
```

Comment:
Since we have typings for options, we can now use them here
```suggestion
			'svelte:options': HTMLProps<'svelte:options', HTMLAttributes>;
```